packages/synapse-react-client/src/components/AddToDownloadCart/AddToDownloadCartButton.tsx

```diff
@@ -37,11 +38,13 @@ const QueryWrapperTableDownloadConfirmation: React.FC<{
   }
 
   return (
-    <QueryWrapper initQueryRequest={initQueryRequest}>
-      <QueryWrapperErrorBoundary>
-        <TableQueryDownloadConfirmation />
-      </QueryWrapperErrorBoundary>
-    </QueryWrapper>
+    <Suspense fallback={<></>}>
```

Comment:
Add suspense to prevent flickering (we should add a better loader fallback, but this will work in a pinch)